packages/devtools_app/lib/src/extensions/embedded/_controller_web.dart

```diff
@@ -95,6 +95,14 @@ class EmbeddedExtensionControllerImpl extends EmbeddedExtensionController
     extensionPostEventStream.add(DevToolsExtensionEvent(type, data: data));
   }
 
+  @override
+  void forceReload() {
+    if (_extensionIFrame.contentWindow != null) {
+      // ignore: unsafe_html, forcing reload.
```

Comment:
Consider changing comment to `// ignore: unsafe_html, forcing reload by resetting the pre-existing IFRAME src`(or something like that) to make it clear why this is safe